### PR TITLE
ANDAUTH-70 Hide Soft Input after clicking on SignIn button

### PR DESCRIPTION
**Pull request by somenath1435**
After clicking on signin button, soft input(keyboard) should be hidden

### CODE REVIEW

**Comment by InsanusMokrassar** [comment]
It will be better, if you will implement something like in this answer:

https://stackoverflow.com/a/59780712

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]
It will be better, if you will implement something like in this answer:

https://stackoverflow.com/a/59780712

**Comment by somenath1435** [comment]
@InsanusMokrassar please check now

File: app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt [hunk 1]
```diff
@@ -306,4 +309,9 @@ class SignInViewFlipper(
             }
         })
     }
+
+    fun View.hideKeyboard() {
```
Comment (InsanusMokrassar) [inline app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt:313]: It is better to use `Context` as a context for this method

Comment (InsanusMokrassar) [reply app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt:313]: Besides, it is better to extract this method into some util file

**Review by InsanusMokrassar — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt (hunk 1 above)
Comment (somenath1435) [reply app/src/main/java/org/xwiki/android/sync/activities/SignInViewFlipper.kt:313]: @InsanusMokrassar In my first commit, I've used the function present in utils folder of this project, you suggested me to take a different approach

**Review by InsanusMokrassar — APPROVED** [review]

**Review by tmortagne — APPROVED** [review]

**Pull request merged**